charts/kashti/templates/js-configmap.yaml

```diff
@@ -1,4 +1,4 @@
-
+{{ $apiServer := include "brigade.apiServer" . }}
```

Comment:
That looks good. Do you still need a port? I can't remember what the current port number is.

Reply:
Yes, a port is still needed; the default is currently `7745`, hence: https://github.com/Azure/brigade-charts/blob/9e31d7b07c9a1699e61942354dd6d779589ef602/charts/kashti/templates/_helpers.tpl#L19 

Therefore, slightly brittle if this port number ever changes; but hopefully the chances of an update in the brigade chart slipping through without a corresponding update for this chart, now that they are both in the same repo, is slim.